workarounds/files/ifup-local

```diff
@@ -6,12 +6,25 @@
 # https://access.redhat.com/solutions/8694
 #
 
+# rahul - all interfaces attached to br-ex are also brought up
+#         when we bring up br-ex
+
 case "$1" in
 
 # for any interface using the ixgbe driver (i.e. em*) turn off Large
 # Receive Offload - see 
 # https://access.redhat.com/support/cases/#/case/01326984
   e*) /sbin/ethtool -K $1 lro off
+  ;;
+  br-ex) DIR="/etc/sysconfig/network-scripts"
+         TEXT="OVS_BRIDGE=br-ex"
+         INTERFACES=$(grep -lr $TEXT $DIR | awk -F "ifcfg-" '{print $NF}')
+
+         for i in "${INTERFACES[@]}"
+         do
+           /sbin/ifup $i
```

Comment:
1. I am not sure what state a nic-card maintains if I bring down an interface manually. Also, if br-ex is up and user tries to do "ifup br-ex", it checks the state of br-ex and if its already up, it does nothing. In this case, the current code will never be exxecuted. I think its a valid case ( some might say that they want the functionality to bring up interface even if its up or not, some might say that if one has manually brought down the interface, then he should himself bring it up). The only thing which we want to avoid with our fix is that ordering should not matter, any interface can be brought up in any order.

2. For ethtool, we do require the fix on compute node side as well. This is required on ethernets of both compute and controller which forms the tunnel endpoints.